nixos/modules/services/misc/mediatomb.nix

```diff
@@ -6,37 +6,97 @@ let
 
   gid = config.ids.gids.mediatomb;
   cfg = config.services.mediatomb;
+  name = cfg.package.pname;
+  pkg = cfg.package;
+  optionYesNo = option: if option then "yes" else "no";
+  # configuration on media directory
+  mediaDirectory = {
+    options = {
+      path = mkOption {
+        type = types.str;
+        description = ''
+          Absolute directory path to the media directory to index
```

Comment:
All the descriptions should add with a period. Unfortunately I didn't find any formal docs to reference here, but it seems to be the de-facto standard (looking at `man configuration.nix`).